packages/backend/src/open_payments/quote/service.ts

```diff
@@ -218,9 +218,10 @@ function calculateQuoteAmountsAfterFees(
   } else {
     // FixedSend
     const fees = quote.fee?.calculate(receiveAmountValue) ?? 0n
-    const exchangeAdjustedFees = BigInt(
-      Number(fees) * quote.estimatedExchangeRate.valueOf()
-    )
+    const estimatedExchangeRate =
+      quote.estimatedExchangeRate || quote.lowEstimatedExchangeRate.valueOf()
```

Comment:
Fallback to lowEstimatedExchangeRate (until https://github.com/interledger/rafiki/issues/2000)